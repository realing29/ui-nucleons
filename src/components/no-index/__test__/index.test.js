import React from 'react';
import { mount } from 'enzyme';
import Index from '../index';

describe('<NoIndex', () => {
  it('should renders correctly', () => {
    const elem = mount(<Index>test<p>test2</p></Index>);
    expect(elem.find('p').text()).toBe('test2');
    expect(elem).toMatchSnapshot();
  });
});
