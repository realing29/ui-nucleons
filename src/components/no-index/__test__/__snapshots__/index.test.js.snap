// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoIndex should renders correctly 1`] = `
<Index>
  <span
    dangerouslySetInnerHTML={
      Object {
        "__html": "<!--noindex-->",
      }
    }
  />
  test
  <p>
    test2
  </p>
  <span
    dangerouslySetInnerHTML={
      Object {
        "__html": "<!--/noindex-->",
      }
    }
  />
</Index>
`;
